src/main/java/seedu/address/model/visittodo/VisitTodo.java

```diff
@@ -0,0 +1,78 @@
+package seedu.address.model.visittodo;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a VisitTodo in the application.
```

Comment:
Perhaps you could explain the distinction between `VisitTodo` and `VisitTask` in the JavaDocs and the rationale for keeping them separate.